tests/Compressor/HtmlCompressorTest.php

```diff
@@ -58,7 +58,7 @@ public function providerMultipleSpaces(): iterable
         ];
         yield [
             "<html>\n    <head>     </head>\n    <body>\n      <p id=\"text\" class=\"foo\">\n        foo\n      </p>  <br />  <ul > <li> <p class=\"foo\">lall</p> </li></ul>\n    </body>\n    </html>",
-            '<html><head> <body><p class=foo id=text> foo </p> <br><ul><li><p class=foo>lall </ul>',
+            '<html><head> <body><p class=foo id=text> foo  <br><ul><li><p class=foo>lall </ul>',
```

Comment:
Is there a specific reason to do this? To me it's unclear why this `</p>` can be removed here, unless I missed something in the specs